config/helm/chart/default/templates/Common/csi/daemonset.yaml

```diff
@@ -52,11 +52,11 @@ spec:
         # - Needs access to the csi socket, needs to read/write to it, needs root permissions to do so.
         # - Needs access to the filesystem of pods on the node, and mount stuff to it,needs to read/write to it, needs root permissions to do so
         # - Needs access to a dedicated folder on the node to persist data, needs to read/write to it.
-      - name: driver
+      - name: server
```

Comment:
annotations should be updated to replace `kubectl.kubernetes.io/default-logs-container: driver` with `kubectl.kubernetes.io/default-container: provisioner`